examples/ui5-js-app/webapp/test/e2e/locators-basic.test.js

```diff
@@ -84,4 +84,79 @@ describe("mixed locators", () => {
         const textViaFluentApi = await browser.asControl(selector).getSelectedItem().getText()
         expect(textViaFluentApi).toEqual("Algeria")
     })
+
+    it("should find the input field on a SearchField", async () => {
+        // will locate the input field
+        const searchFieldSelectorInput = {
+            selector: {
+                controlType: "sap.m.SearchField",
+                viewName,
+                interaction: "focus"
+            }
+        }
+        const placeholder = await browser.asControl(searchFieldSelectorInput)
+        const placeholderText = await placeholder.getPlaceholder()
+        expect(placeholderText).toEqual("Search...")
+    })
+
+    it.only("should operate a SearchField via OPA5-compatible press- and focus-interaction", async () => {
```

Comment:
The `only` is not necessary anymore, right? ;) 
```suggestion
    it("should operate a SearchField via OPA5-compatible press- and focus-interaction", async () => {
```

Reply:
well spotted, thx, and done!